Resources/views/Collector/migrations.html.twig

```diff
@@ -0,0 +1,175 @@
+{% extends app.request.isXmlHttpRequest ? '@WebProfiler/Profiler/ajax_layout.html.twig' : '@WebProfiler/Profiler/layout.html.twig' %}
+
+{% block toolbar %}
+    {% set profiler_markup_version = profiler_markup_version|default(1) %}
+
+    {# this is the content displayed as a panel in the toolbar #}
+    {% set icon %}
+        {% if profiler_markup_version == 1 %}
+            {% set color = collector.executedUnavailableMigrations > 0 ? 'red' : '' %}
+            {% if color is empty %}
+                {% set color = 'yellow' %}
```

Comment:
btw, you don't handle colors in the profiler v2 for Symfony 2.8+, which looks inconsistent